src/components/Overlay/index.tsx

```diff
@@ -281,14 +281,14 @@ const OverlayPage = (props) => {
             width={width}
             height={height}
             alt={`${block.type} dev screenshot`}
-            src={`/images/dev/${
-              block.type === 'spectator' ? 'playing' : block.type
-            }.png`}
+            src={`/images/dev/21-9-playing.png`}
           />
         )}
       </AnimatePresence>
     </>
   )
 }
 
+OverlayPage.whyDidYouRender = true
```

Comment:
This is new to me. Would you mind enlightening me on this?
generally mutation like this is an anti-pattern in React, but I feel like there's some reasoning behind this. Maybe for optimization .. but then, how does it work?